app/scripts/lib/encryptor-factory.test.ts

```diff
@@ -0,0 +1,170 @@
+import * as browserPassworder from '@metamask/browser-passworder';
+import { encryptorFactory } from './encryptor-factory';
+
+jest.mock('@metamask/browser-passworder');
+
+const mockIterations = 100;
+const mockPassword = 'password';
+const mockData = 'data';
+
+describe('encryptorFactory', () => {
```

Comment:
Nit: As a precaution against accidentally writing tests that depend on each other, I'd recommend resetting mocks in between each test, e.g. 

```suggestion
describe('encryptorFactory', () => {

  afterEach(async () => {
    jest.resetAllMocks();
  });
```

Normally in our libraries the `resetMocks` setting takes care of this automatically, but that's not yet enabled in this repo.